production_configs/20240917_v0.10.12_src15_dec_min_2924_tuned/README.md

```diff
@@ -3,7 +3,11 @@
 I need this production to analyse the LST-1 data of src15 (see https://www.lst1.iac.es/wiki/index.php/MC_analysis_and_IRF_production#lstmcpipe_productions) with dec 29.24 deg. The median of the NSB standard deviation for this dataset is approximately 2.308 p.e.
 
 '''
+<<<<<<< HEAD
+lstmcpipe_generate_config PathConfigAllSkyFullDL1ab --dec_list dec_min_2924 --prod_id 20240917_v0.10.12_src15_dec_min_2924_tuned --kwargs source_prod_id=20240131_allsky_v0.10.5_all_dec_base
+=======
 lstmcpipe_generate_config PathConfigAllSkyFullDL1ab --dec_list dec_min_2924 --prod_id 20240917_v0.10.12_src15_dec_min_2924_tuned --kwargs source_prod_id=20240917_v0.10.12_src15_dec_min_2924_tuned
+>>>>>>> 8c85c8a6cd11e8bbaf70eebfbb9ae4ab9215c1ed
```

Comment:
you need to handle the git merge here by keeping only the part you want (here the first one)

Reply:
Ok, thank you